README.md

```diff
@@ -20,7 +19,91 @@ That's it! All other dependencies are included in the Docker containers that dka
 ```bash
 ln -s /my/dktl/location/bin/dktl.sh ~/bin/dktl
```

Comment:
Mac System Integrity Protection does not allow symlink to `~/bin/dktl`, so using `/usr/local/bin/dktl`